src/domain/event/utils.ts

```diff
@@ -131,3 +134,20 @@ export const isFutureEvent = (eventData: EventQuery | undefined) =>
 
 export const isEditableEvent = (eventData: EventQuery | undefined) =>
   isFutureEvent(eventData);
+
+export const hasOccurrences = (event: EventFieldsFragment) => {
+  return Boolean(event.pEvent?.occurrences.edges.length);
+};
+
+export const hasComingOccurrences = (event: EventFieldsFragment) => {
+  let hasComingItems = false;
+
+  forEach(event.pEvent?.occurrences.edges, (edge) => {
+    if (edge?.node?.startTime && isFuture(new Date(edge?.node?.startTime))) {
+      hasComingItems = true;
+      return false;
+    }
+  });
+
+  return Boolean(hasComingItems);
```

Comment:
Maybe a little bit redundant to have Boolean() here because hasComingItems is already a boolean and the name implies that also